src/frontend/src/handler/show.rs

```diff
@@ -106,6 +108,17 @@ fn schema_or_default(schema: &Option<Ident>) -> String {
         .map_or_else(|| DEFAULT_SCHEMA_NAME.to_string(), |s| s.real_value())
 }
 
+fn schema_or_search_path<'a>(
```

Comment:
You can follow the example here:
https://github.com/risingwavelabs/risingwave/blob/0c8371ab8aae97e2cf782741457d553c295bf675/src/frontend/src/binder/relation/table_or_source.rs#L148

Note that `USER_NAME_WILD_CARD` should be handle separately.